scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -197,12 +200,13 @@
     true)
 
   (pending-offers [this pool-name]
-    (if (or (= pool-name (config/default-pool)) ; TODO(pschorf): Support pools
-            (= pool-name "no-pool"))
-      (let [nodes @current-nodes-atom
-            pods @all-pods-atom]
-        (generate-offers this nodes pods (controller/starting-namespaced-pod-name->pod this)))
-      []))
+    (let [nodes @current-nodes-atom
+          pods @all-pods-atom
+          starting-instances (controller/starting-namespaced-pod-name->pod this)
+          offers-all-pools (generate-offers this nodes pods starting-instances)]
```

Comment:
Can you add a `TODO` to make this calculation pool-specific instead of generating all of them and throwing them away?
I think it's fine for now, though.

Reply:
Yeah, and good reminder that we should have a TODO. I'll open a JIRA to track too.

Reply:
Can you add the TODO?

Reply:
Its there a couple of lines lower.